lib/puppetlabs_spec_helper/rake_tasks.rb

```diff
@@ -193,14 +193,21 @@ def revision(scm, target, ref)
   FileUtils.rm_rf("pkg/")
 end
 
-desc "Check puppet manifests with puppet-lint"
-task :lint do
-  require 'puppet-lint/tasks/puppet-lint'
-  PuppetLint.configuration.relative = true
-  PuppetLint.configuration.disable_class_inherits_from_params_class
-  PuppetLint.configuration.ignore_paths ||= []
-  PuppetLint.configuration.ignore_paths << "spec/fixtures/**/*.pp"
-  PuppetLint.configuration.ignore_paths << "pkg/**/*.pp"
+require 'puppet-lint/tasks/puppet-lint'
+# Must clear as it will not override the existing puppet-lint rake task since we require to import for
+# the PuppetLint::RakeTask
+Rake::Task[:lint].clear
+# Relative is not able to be set within the context of PuppetLint::RakeTask
```

Comment:
For the record, this is fixed in https://github.com/rodjek/puppet-lint/commit/c033c3739599161b8164e6d9738d0e0b3c8f1d72 (to be 1.1.1).